### PR TITLE
feature/kitosudv 5223 org unit roles

### PR DESCRIPTION
**Pull request by JacobBorchMadsen**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [ ] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [ ] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Core.DomainModel/Organization/OrganizationRight.cs [hunk 1]
```diff
@@ -1,4 +1,6 @@
-﻿namespace Core.DomainModel.Organization
+﻿using Core.DomainModel.ItContract;
```
Comment (JacobBorchMadsen) [inline Core.DomainModel/Organization/OrganizationRight.cs:1]: remove this

File: Core.ApplicationServices/Organizations/Write/OrganizationUnitWriteService.cs [hunk 2]
```diff
@@ -7,6 +7,7 @@
 using System;
 using Core.ApplicationServices.Extensions;
 using Core.ApplicationServices.Model.Shared;
+using Core.DomainServices.Role;
```
Comment (a-naskret) [inline Core.ApplicationServices/Organizations/Write/OrganizationUnitWriteService.cs:10]: unused using?

File: Presentation.Web/Models/API/V2/Request/OrganizationUnit/DeleteOrganizationUnitRoleAssignmentRequestDTO.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
```
Comment (a-naskret) [inline Presentation.Web/Models/API/V2/Request/OrganizationUnit/DeleteOrganizationUnitRoleAssignmentRequestDTO.cs:4]: usings can be deleted

**Review by a-naskret — APPROVED** [review]
Looks good if green

**Pull request merged**